hr_holidays_meeting_name/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+========================
+HR holidays meeting name
+========================
+
+This module was written to allow to personnalize meeting name generated by leaves with the leave type.
```

Comment:
This module allows to leave generated meeting name personalization according to the leave type.